packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -443,21 +510,21 @@ export class Table extends Construct {
   }
 
   private buildAutoScalingRole(roleResourceName: string) {
-    const autoScalingRole = new Role(this, roleResourceName, {
-      assumedBy: new ServicePrincipal('application-autoscaling.amazonaws.com')
+    const autoScalingRole = new iam.Role(this, roleResourceName, {
+      assumedBy: new iam.ServicePrincipal('application-autoscaling.amazonaws.com')
     });
-    autoScalingRole.addToPolicy(new PolicyStatement(PolicyStatementEffect.Allow)
+    autoScalingRole.addToPolicy(new iam.PolicyStatement(iam.PolicyStatementEffect.Allow)
```

Comment:
Is `Allow` even necessary?

Reply:
Not really but makes it a bit more explicit